### ISSUE TITLE
Bug Admin /functions/configuration_checks.php: str_pos should be strpos 

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#2837]
In admin /includes/functions/configuration_checks.php 
I believe the two instances of str_pos should be strpos:
```
switch (true) {
          case (str_pos($data['error'], 'TEXT_MIN_ADMIN') === 0):
            $error_msg = TEXT_MIN_GENERAL_ADMIN;
            break;
          case (str_pos($data['error'], 'TEXT_MAX_ADMIN') === 0);
            $error_msg = TEXT_MAX_GENERAL_ADMIN;
            break;
          default:
            $error_msg = TEXT_DATA_OUT_OF_RANGE;
        }
```

**Comment by drbyte** [comment]
Correct

**Comment by drbyte** [comment]
Closed in #2845
